clang/test/Driver/sycl-unsupported.cpp

```diff
@@ -31,8 +31,6 @@
 // RUN:  | FileCheck %s -DARCH=spir64 -DOPT=-fcs-profile-generate \
 // RUN:    -DOPT_CC1=-fprofile-instrument=csllvm \
 // RUN:    -check-prefixes=UNSUPPORTED_OPT_DIAG,UNSUPPORTED_OPT
-// RUN: %clangxx -fsycl -forder-file-instrumentation -### %s 2>&1 \
```

Comment:
How is this change related to this PR?

Reply:
Same comment as above - just fixing LIT tests in general.